sdks/python/apache_beam/ml/inference/sklearn_inference.py

```diff
@@ -74,8 +91,12 @@ def load_model(self) -> BaseEstimator:
     return _load_model(self._model_uri, self._model_file_type)
 
   def run_inference(
-      self, batch: Sequence[numpy.ndarray], model: BaseEstimator,
-      **kwargs) -> Iterable[PredictionResult]:
+      self,
+      batch: Sequence[numpy.ndarray],
+      model: BaseEstimator,
+      inference_args: Optional[Dict[str, Any]] = None
+  ) -> Iterable[PredictionResult]:
+    _validate_inference_args(inference_args)
```

Comment:
(not urgent). This validation will  happen at pipeline execution, right? Can it be done at construction time to reduce the feedback loop?

Reply:
Yes. I've filed this issue https://github.com/apache/beam/issues/21894 as something to do later.